api/services/images/v1/images.pb.go

```diff
@@ -436,7 +436,7 @@ type ListImagesRequest struct {
 	// filters. Expanded, images that match the following will be
 	// returned:
 	//
-	//   filters[0] or filters[1] or ... or filters[n-1] or filters[n]
+	//	filters[0] or filters[1] or ... or filters[n-1] or filters[n]
```

Comment:
no mercy from linter... I think we should update proto first and regenerate it.

Reply:
Yup. Updated protobufs in https://github.com/containerd/containerd/pull/7254/commits/68bae258a3587dec6c7c07b8083d4c8bc540a6e2